pkg/log.go

```diff
@@ -51,8 +51,8 @@ func SetLogFuncCall(b bool) {
 
 // SetLogger sets a new logger.
 // Deprecated: use github.com/astaxie/beego/logs instead.
-func SetLogger(adaptername string, config string) error {
-	return logs.SetLogger(adaptername, config)
+func SetLogger(adaptername string, fmtFunc logs.FormatterFunc, config string) error {
```

Comment:
I think we should add a new method named 
```go
SetLoggerWithOpts(adaptername string,  config string, opts... common.KV)
```
We could use KV defined in pkg/common or defining similar structure in log module.

In most cases, users don't want to custom anything, so they can use old method `SetLoggger`

Reply:
I agree that most users would not want to set a formatter and `SetLoggerWithOpts` would be a better idea.